Snowdog_Components/components/Molecules/image/image--ratio.hbs

```diff
@@ -1,9 +1,13 @@
 <div
-    class="lazyload-wrapper {{ wrapperClass }}"
+    class="lazyload-wrapper ratio-container"
```

Comment:
please still keep `{{ wrapperClass }}` here. this component can be reused in other components and `wrapperClass` would allow adding some additional classes in `.config.js` of these components if needed